workflow/usage.rst

```diff
@@ -255,8 +255,9 @@ order:
     * ``workflow.[workflow name].transition.[transition name]``
 
 ``workflow.enter``
-    The object entered a new place. This is the first event where the object
-    is marked as being in the new place.
+    The object is about to enter a new place. This is the event where the object is
+    going to be marked as being in the new place.
```

Comment:
Actually, this is the event **before** the subject (everywhere in the code I used subject and not object, I think it's better if the doc reflects that) is going to be updated

Reply:
Updated as:
```
    The subject is about to enter a new place. This is the event triggered before the subject is
    going to be updated as being in the new places.
    Please notice: the marking of the subject is not yet updated with the new places.
```